playground/mocks/idp/idx/data/select-factor-authenticate-authenticators.json

```diff
@@ -0,0 +1,166 @@
+    "version": "1.0.0",
+    "expiresAt": "2020-05-05T23:34:09.000Z",
+    "step": "AUTHENTICATE",
+    "intent": "LOGIN",
+    "remediation": {
+        "type": "array",
+        "value": [
+            {
+                "rel": [
+                    "create-form"
+                ],
+                "name": "select-factor-authenticate",
+                "href": "http://localhost:3000/idp/idx/challenge",
+                "method": "POST",
+                "accepts": "application/vnd.okta.v1+json",
+                "value": [
+                    {
+                        "name": "authenticator",
+                        "type": "object",
+                        "options": [
+                            {
+                                "label": "Okta Password",
+                                "form": {
+                                    "value": [
+                                        {
+                                            "name": "id",
+                                            "required": true,
+                                            "value": "autwa6eD9o02iBbtv0g3",
+                                            "mutable": false
+                                        },
+                                        {
+                                            "name": "methodType",
+                                            "required": false,
+                                            "value": "password",
+                                            "mutable": false
+                                        }
+                                    ]
+                                }
+                            },
+                            {
+                                "label": "Security Key or Biometric Authenticator (FIDO2)",
```

Comment:
I did not see these options show up in your demo video. Are we handling the selection of any authenticator with this PR or just the password authenticator ?

Reply:
try to play with a few examples that we know the api structure.